node/core/dispute-coordinator/src/import.rs

```diff
@@ -56,10 +55,9 @@ impl<'a> CandidateEnvironment<'a> {
 	/// Return: `None` in case session is outside of session window.
 	pub fn new(
 		keystore: &LocalKeystore,
-		session_window: &'a RollingSessionWindow,
+		session: &'a SessionInfo,
```

Comment:
`SessionInfo` is obtained outside this function and passed directly. Otherwise it should be async.

Reply:
Okay, that's a very valid point.

Reply:
Fixed